src/components/Navbar/SettingsDrawer/Section/index.tsx

```diff
@@ -6,8 +6,10 @@ import Title from './Title';
 
 import Separator, { SeparatorWeight } from '@/dls/Separator/Separator';
 
-const Section = ({ children }) => (
-  <div className={styles.section}>
+interface SectionProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+const Section = ({ children, ...props }: SectionProps) => (
```

Comment:
Ditto: I prefer to explicitly show which props we are passing exactly since it's very vague looking at the component and we have to go check the usage to know what was passed etc.